server/accounts.go

```diff
@@ -2815,10 +2815,6 @@ func (*resolverDefaultsOpsImpl) Reload() error {
 func (*resolverDefaultsOpsImpl) Close() {
 }
 
-func (*resolverDefaultsOpsImpl) Store(_, _ string) error {
```

Comment:
Should not remove this one. This is the "base" implementation that has all interface's method implemented - even if returning errors. Other implementations that embed this implementation will either override the method they chose or fallback to the base implementation.
I agree that the error text could be changed though.

Reply:
I added the default `Store` method but changed a bit in the error text.